.ci/configure_crio_for_kata.sh

```diff
@@ -34,14 +34,14 @@ if [ "$minor_crio_version" -ge "12" ]; then
 	kata_configured=$(grep -q $kata_flag $crio_config_file; echo "$?")
 	if [[ $kata_configured -ne 0 ]]; then
 		sudo sed -i '/\/run\/runc/a [crio.runtime.runtimes.kata]' "$crio_config_file"
-		sudo sed -i '/crio\.runtime\.runtimes\.kata\]/a runtime_path = "/usr/local/bin/kata-runtime"' "$crio_config_file"
-		sudo sed -i '/kata-runtime"/a runtime_root = "/run/vc"' "$crio_config_file"
-		sudo sed -i '/\/run\/vc/a runtime_type = "oci"' "$crio_config_file"
+		sudo sed -i '/crio\.runtime\.runtimes\.kata\]/a runtime_path = "/usr/local/bin/containerd-shim-kata-v2"' "$crio_config_file"
+		sudo sed -i '/containerd-shim-kata-v2"/a runtime_root = "/run/vc"' "$crio_config_file"
+		sudo sed -i '/\/run\/vc/a runtime_type = "vm"' "$crio_config_file"
 	fi
 else
 	echo "Configure runtimes for trusted/untrusted annotations"
```

Comment:
I would say that we can remove this block as trusted/untrusted is no longer used for our CI.